Grinnell-DB-iOS/FormViewController.m

```diff
@@ -1,23 +1,17 @@
 #import <Crashlytics/Crashlytics.h>
-#import <MBProgressHUD.h>
 #import <Reachability.h>
 
 #import "FormViewController.h"
+#import "Grinnell_DB_iOS-Swift.h"
 #import "OptionViewController.h"
 #import "Person.h"
-
 #import "ResultsViewController.h"
-#import "Grinnell_DB_iOS-Swift.h"
-
-@interface FormViewController ()
-
-@end
 
 @implementation FormViewController
-@synthesize lastNameField, firstNameField, usernameField, phoneField, campusAddressField, homeAddressField, majorField, concentrationField, sgaField, hiatusField, classField, facStaffField, keyboardControls, textFieldIdentifier, myPickerView, concentrationArray, sgaArray, facStaffArray, hiatusArray, classArray, majorsArray, searchResults, onCampusBool, notFirstRun, statesArray, stateBeforeSettings;
+@synthesize lastNameField, firstNameField, usernameField, phoneField, campusAddressField, homeAddressField, majorField, concentrationField, sgaField, hiatusField, classField, facStaffField, textFieldIdentifier, myPickerView, concentrationArray, sgaArray, facStaffArray, hiatusArray, classArray, majorsArray, searchResults, onCampusBool, notFirstRun, statesArray, stateBeforeSettings;
```

Comment:
I think property synthesis happens automatically as of like XCode 4. I think you should delete this unnecessary, unnecessarily long line entirely